build/ci-build.yml

```diff
@@ -93,8 +93,15 @@ stages:
     jobs:
       - job: RunIntegrationTests
         displayName: 'Run integration tests'
+        strategy:
+          maxParallel: 1
+          matrix:
+            linux:
+              imageName: 'ubuntu-latest'
+            windows:
+              imageName: 'windows-latest'
```

Comment:
Apparently, we can't use variables in strategy.matrix variable definitions... too bad.

Reply:
What exactly do you mean ?  Do you mean you want to refer to the `Vm.Linux.Image` and `Vm.Windows.Image` variables ?

Is this also not possible via [Template Expression Syntax](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#template-expression-syntax) ? Like this:

```yaml
${{ variables['Vm.Linux.Image']}}
```

Reply:
I'll try that, but don't have much hope 😅 .

Reply:
Aha, yeah, that did it. Forgot that template expression variables were initialized at runtime, and would therefor do the trick. Thx!